.travis.yml

```diff
@@ -15,8 +15,6 @@
 
 language: java
 
-sudo: false
```

Comment:
What is the reason for removing this guy?

This is an important `switch`, that makes sure all builds are using [container-based infrastructure](https://docs.travis-ci.com/user/migrating-from-legacy/) on TravisCI.

I spent tremendous about of time earlier this year (i.e [ZEPPELIN-776](https://issues.apache.org/jira/browse/ZEPPELIN-776)), fighting non-stable CI that was due to some builds being allocated on non-container infra and networking there is different. 

So without a good reason, I strongly recommend putting it back.